README.md

```diff
@@ -102,6 +102,77 @@ Where the `format`s available are:
 - `spdx-json`: A JSON report conforming to the [SPDX 2.2 JSON Schema](https://github.com/spdx/spdx-spec/blob/v2.2/schemas/spdx-schema.json).
 - `table`: A columnar summary (default).
 
+## Private Registry Authentication
+
+Syft uses the [authn](https://github.com/google/go-containerregistry/tree/main/pkg/authn) library to solve for private registry access. 
```

Comment:
My two cents: "authn" isn't a library, it's a package within the "google/go-containerregistry" (nicknamed "GGCR") library. Curious for others' thoughts.

Reply:
updated to package